.github/workflows/amplify_canaries.yaml

```diff
@@ -46,6 +50,26 @@ jobs:
       - name: Build Canary (Android)
         run: build-support/build_canary.sh apk
 
+      - name: Log failing builds
+        if: ${{ failure() }}
+        uses: ./.github/composite_actions/log_metric
+        with:
+          role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+          aws-region: ${{ secrets.AWS_REGION }}
+          metric-name: BuildCanaryTestFailure
+          value: 1
+          dimensions: channel=${{ matrix.channel }},label=CanaryBuildFailures
```

Comment:
Did you add these labels since last review? I don't understand why they would necessary. The metric name already groups values.

Reply:
+1

Reply:
Missing platform too

Reply:
I'm a bit confused by this comment then: 
https://github.com/aws-amplify/amplify-ci-support/pull/146#discussion_r1240541017

It seems to suggest we should add a label property.

Reply:
Here's an internal link to the Slack message from last week in which I mentioned why we currently do not track platform for the build step: https://amzn-aws.slack.com/archives/C02JC797CGK/p1687551527260999

We can split up the build step but perhaps that's not necessary yet.

Reply:
Thanks, removed the label from dimensions.